Installation/include/CGAL/config.h

```diff
@@ -623,6 +623,7 @@ using std::max;
 #  include <unordered_set>
 #  include <unordered_map>
 #  include <functional>
+#  include <CGAL/result_of.h>
```

Comment:
Why do you need that?! I disagree. That header will pull too much of Boost into all headers of CGAL.